### PR TITLE
Better home page routing

### PR DESCRIPTION
**Pull request by hardins01**
This is a quick one to make a home page that actually acts as a home page, being the main landing page that you go to before logging in. This means I had to move the functionality that used to be the `HomePageComponent` into the `ProjectsPageComponent`. `HomePageComponent` is now a basic page with the cuttlefish logo and a button to navigate to the login page. `ProjectsPageComponent` now shows the list of projects.

I also made it so that the nav bar doesn't show up until the user gets to the projects page, because users shouldn't be able to use it until they're logged in. I followed Hess's suggestions with the nav bar to make the list of projects show up in a dropdown, as well as make the logo at the top route to the projects dashboard.

The last tiny change was to update the favicon of the site to the cuttlefish logo.

### CODE REVIEW

File: ClientApp/src/index.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <base href="/" />
 
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <link rel="icon" type="image/x-icon" href="favicon.ico" />
+    <link rel="icon" type="image/png" href="assets/cuttlefish_logo.png" />
```
Comment (NathanJesudason) [inline ClientApp/src/index.html]: As discussed, for the favicon it needs to be a .ico file

**Review by NathanJesudason — APPROVED** [review]

**Comment by hardins01** [comment]
I'm gonna update this pr a bit with the nav bar routing feedback hess gave

**Comment by hardins01** [comment]
Oops should close this until I'm ready to reopen

**Comment by hardins01** [comment]
Ready for a PR now!

File: ClientApp/src/index.html (hunk 1 above)
Comment (hardins01) [reply ClientApp/src/index.html]: Done

**Review by NathanJesudason — APPROVED** [review]

**Pull request merged**